### PR TITLE
feat(eas): Allow EAS Search operation in all parts of a message

### PR DESCRIPTION
**Pull request by tfux**
For bug #5206.

### CODE REVIEW

File: ActiveSync/SOGoActiveSyncDispatcher.m [hunk 1]
```diff
@@ -3115,12 +3115,16 @@ - (EOQualifier *) _qualifierFromMailboxSearchQuery: (id <DOMElement>) theDocumen
 	}
 
       notDeleted = [EOQualifier qualifierWithQualifierFormat: @"(not (flags = %@))", @"deleted"];
-      subjectQualifier = [EOQualifier qualifierWithQualifierFormat: [NSString stringWithFormat: @"(%@ doesContain: '%@')", @"subject", query]];
-      senderQualifier = [EOQualifier qualifierWithQualifierFormat: [NSString stringWithFormat: @"(%@ doesContain: '%@')", @"from", query]];
 
-      orQualifier = [[EOOrQualifier alloc] initWithQualifiers: subjectQualifier, senderQualifier, nil];
+      if ([[SOGoSystemDefaults sharedSystemDefaults] easSearchInBody]) {
+         searchQualifier = [EOQualifier qualifierWithQualifierFormat: [NSString stringWithFormat: @"(%@ doesContain: '%@')", @"body", query]];
```
Comment (extrafu) [inline ActiveSync/SOGoActiveSyncDispatcher.m:3120]: Why not use "TEXT" instead of "BODY" here? It would also search in headers.

**Comment by tfux** [comment]
Made an update to use argument TEXT instead of BODY for IMAP SERACH.

**Comment by cgx** [comment]
> Made an update to use argument TEXT instead of BODY for IMAP SERACH.

Good, thanks. Can you also squash your two commits?

**Comment by tfux** [comment]
Do you need anything more from my side?

**Comment by jacotec** [comment]
@tfux 
Unfortunately this fix seems not to work. I'm still getting only results where the search word is in the email subject, but no results where the search word is in the email body when doing a search on my iPhone/iPad which are connected via Active Sync.

Was this tested on these devices?

**Comment by cgx** [comment]
@jacotec did you set `SOGoEASSearchInBody` to `YES`?

**Comment by jacotec** [comment]
@cgx Oh, no ... I did not know. I've added this to my sogo.conf and restarted it, but it still does not find the word in the bodies. Any more ideas what might be missing?

**Pull request merged**